voluncheer/profiles/test_views.py

```diff
@@ -1,10 +1,10 @@
 from django.test import TestCase
 from django.urls import reverse
 
-from profiles.models import Organization
-from profiles.models import User
-from profiles.models import UserType
-from profiles.models import Volunteer
+# from profiles.models import Organization
```

Comment:
I think we should remove the imports if we decide to not use them?

Reply:
This isn't my addition, so I am not sure what they were intending. I agree with you, but I don't want to mess with someone else's code.

Reply:
We should definitely remove this before merging though. It's OK if this PR is still a work in progress but upon submission we should have commented out code.

For the rest of this review I'll assume all of the commented code falls into this.